test/cppsuite/src/main/database_operation.cpp

```diff
@@ -130,10 +130,27 @@ database_operation::background_compact_operation(thread_worker *tc)
     logger::log_msg(
       LOG_INFO, type_string(tc->type) + " thread {" + std::to_string(tc->id) + "} commencing.");
 
-    /* This needs to be executed only once in the workload. */
-    const std::string compact_cfg(
-      "background=true,free_space_target=" + std::to_string(tc->free_space_target_mb) + "MB");
-    testutil_check(tc->session->compact(tc->session.get(), nullptr, compact_cfg.c_str()));
+    bool enabled = false;
```

Comment:
Can we leave the default implementation as it is and have this custom implementation in your test? You can override this function.

Reply:
Yeah, can do.

Reply:
Fixed https://github.com/wiredtiger/wiredtiger/pull/9691/commits/b21630dc0d16a3e94d7d729cc9d519ab60c84c7e